rna/models.py

```diff
@@ -43,7 +55,7 @@ class Release(TimeStampedModel):
     is_public = models.BooleanField(default=False)
     bug_list = models.TextField(blank=True)
     bug_search_url = models.CharField(max_length=2000, blank=True)
-    system_requirements = models.TextField(blank=True)
+    system_requirements = models.ForeignKey(SystemRequirements, on_delete=models.CASCADE)
```

Comment:
This would delete all of the current system requirements data. I think we should keep this field, add a new one to use as a foreign key to the new `SystemRequirements` table, and do like Sylvestre says in the bug and have some logic that says "if there is text in system_requirements, otherwise use the foreign key, otherwise fall back to some default". We could change the name of the field to e.g. `_system_requirements`, and have a property on the class called `system_requirements` that would use that logic.

Also we need a schema migration to cover this change and the addition of the `SystemRequirements` table.